### ISSUE TITLE
Inconsistency between Song and Pattern Editor

### ISSUE THREAD

**Issue by tilllt** [polyend/TrackerIssues#584]
When i am editing a Song and my Cursor is on a certain Pattern, switching to the Pattern Editor i would expect to go into editing Mode for the pattern that was last selected in Song mode. Its not: its the Pattern that was selected before in Pattern mode.

**Comment by doelman** [comment]
Yeah I noticed this as well. I think it used to behave like that before 1.3

**Comment by MetabaronCZ** [comment]
Yes, this behavior is unintuitive.

**Comment by clickysteve** [comment]
This is arguably a bug, rather than an enhancement request.
